src/components/QuranReader/TranslationView/TranslationViewCell.module.scss

```diff
@@ -16,6 +16,24 @@
   padding-top: var(--spacing-medium);
   position: relative;
 }
+
+.splitView {
+  @include breakpoints.tablet {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row-reverse;
+
+    & > .verseTranslationsContainer {
+      margin-top: calc(1.2 * var(--spacing-medium));
```

Comment:
a) this will have to be a logical property

b) should this value change based on the arabic font size? to better align it with the text